modules/ROOT/partials/couchbase-operator-2.7.1-release-notes.adoc

```diff
@@ -0,0 +1,39 @@
+[#release-271]
+== Release 2.7.1 (August 2025)
+
+Couchbase Operator 2.7.1 was released in August 2025.
+This maintenance release contains fixes to issues.
+
+[#fixed-issues-v271]
+== Fixed Issues
+
+
+[#table-fixed-issues-v271,cols="25,66"]
+
+|===
+|Issue | Description
+
+
+
+| https://jira.issues.couchbase.com/browse/K8S-4038/[K8S-4038]
+
+a| In prior versions of Couchbase Operator, it was possible to set invalid values for the Indexer’s Replica Count.
```

Comment:
We used to have a "Summary: " prefix to each description - if this is not suitable for the current style, then please remove it from the existing Release Notes too.